### DISCUSSION TITLE
Search tabs and switch

### DISCUSSION THREAD

**Discussion by mohitWrangler** [doublecmd/doublecmd#1181] [General]
Is it possible to search for a tab among all saved tabs and switch to it after selection? I use this feature a lot in [fman](https://fman.io/) file manager.

**Comment by edmundlaugasson** [comment]
Looking also tabs hotlist search and switch feature.
Currently there is a shortcut key CTRL+SHIFT+A, that opens the hotlist of opened tabs (like in Brave web browser).  Then you can open either with _keyboard arrow keys + Enter_ or use mouse to choose certain tab in that tabs hotlist and click on it in order to open it. Currently there in tabs hotlist is search missing. 

**Hereby I propose to add same search feature like there is in Directory Hotlist**:
<img width="1241" height="1163" alt="directory-hotlist-search-enabling" src="https://github.com/user-attachments/assets/144570b8-9042-4c09-8a39-4a351544dddb" />
So, basically _Configuration->Options...->Layout->Tree View Menu->With Directory Hotlist->With the menu and internal command_ is turned on, then while pressing CTRL+D you can see that search already exist in DoubleCMD.  Otherwise there is no search field on top of directory hotlist.
<img width="560" height="239" alt="directory-hotlist-search" src="https://github.com/user-attachments/assets/12910a99-b6c5-4d74-95ac-ec116b78ca23" />
Current images are made in DoubleCMD v1.1.26 gamma (build 2025/06/07) in Linux.

So - same search could be also used in tabs hotlist, which is currently missing. There is no need to reinvent the wheel - that search already exists in DoubleCMD source code at directory hotlist, just adapt it to tabs hotlist and that would be sufficient.

Whenever there are lots of tabs opened, would also be reasonable resize that hotlist window size, so it won't occupy too much or even whole screen. According used screen resolution, the tabs hotlist menu size could be adaptive - while smaller resolution, also tabs hotlist menu could be smaller and vice versa.
